src/test/groovy/org/shipkit/internal/gradle/AutoDiscoverGradlePluginsPluginTest.groovy

```diff
@@ -0,0 +1,48 @@
+package org.shipkit.internal.gradle
+
+import com.gradle.publish.PluginBundleExtension
+import testutil.PluginSpecification
+
+class AutoDiscoverGradlePluginsPluginTest extends PluginSpecification {
+
+    public static final String META_INF_GRADLE_PLUGINS = 'src/main/resources/META-INF.gradle-plugins'
+
+    def "Apply"() {
+        expect:
+        project.plugins.apply(AutoDiscoverGradlePluginsPlugin)
+    }
+
+    def "DiscoverGradlePluginPropertyFiles"() {
```

Comment:
Our naming convention for Spock test methods is:
 - all lowercase
 - using spaces

Welcome to Groovy world - you can use spaces in test method names :) This will improve readability of tests.